dynamic-demo-plugin/src/components/Modals/ModalPage.tsx

```diff
@@ -115,6 +115,7 @@ const TestOverlayComponent: OverlayComponent<TestOverlayComponentProps> = ({
   );
 };
 
+// @ts-ignore
```

Comment:
if we really need TS to ignore this, prefer `ts-expect-error` as it will alert you when the statement is no longer needed

if possible it would be a better solution to fix the type error instead of suppressing it here

```suggestion
// @ts-expect-error - [insert reason for ts-expect-error here]
```